src/app/about/About.module.css

```diff
@@ -0,0 +1,30 @@
+.about {
+    /* styles */
+    text-align: center;
+}
+
+.about__image_container {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 2rem;
+    margin-top: 2rem;
+
+}
+
+.about__title {
+    margin-bottom: 5rem;
+    font-size: 3rem;
+    font-family: fantasy;
+}
+
+.about__description_container {
+    width: 100%;
+    
+}
+
+.about__description {
+    margin: 0 auto;
+    max-width: 1000px;
+    font-size: 1rem;
+    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+}
```

Comment:
[Nit]: Newlines and end of files - check editor settings for this one and look for the red symbol on GitHub. This one dates back to file concatenation (e.g. header files) if you're interested.